source/includes/changelogs/releases/100.9.1.rst

```diff
@@ -5,12 +5,20 @@
 
 *Released 2023-11-09*
 
-This version updates external dependencies and fixes an issue where
-mongodump could change the ordering of fields in a view definition.
+This version updates external dependencies and fixes an issue where mongodump 
+could change the ordering of fields in a view definition (:issue:`TOOLS-3411`). 
+View pipelines and schema validation are the only collection options sensitive 
+to field ordering. Depending on the contents of the view or validator options, 
+the following can change:
+
+- The set of results returned by a view, or
+- The criteria determining which documents are accepted by a validator.
```

Comment:
[suggestion][ style guide](https://www.mongodb.com/docs/meta/style-guide/writing/clarify-gerunds-participles/#clarify-gerunds-and-participles) - clarify gerunds and participles
```suggestion
- The criteria that determines which documents are accepted by a validator.
```

Reply:
Done, thank you!